CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-### 0.7.1 (Next)
+### 0.8.1 (Next)
```

Comment:
We want 0.8.0 to be next since there're interface changes, so just change 0.7.1 to 0.8.0, no need to have a 0.8.1 here.